Referenced by issue blender/blender-addons#60934, Undo crashes when calling bpy.ops.ed.undo_history(index=0) from Python in background mode
